sycl/doc/UsersManual.md

```diff
@@ -57,6 +57,46 @@ and not recommended to use in production environment.
     * intel_gpu_kbl, intel_gpu_9_1_9 - Kaby Lake Intel graphics architecture
     * intel_gpu_skl, intel_gpu_9_0_9 - Skylake Intel graphics architecture
     * intel_gpu_bdw, intel_gpu_8_0_0 - Broadwell Intel graphics architecture
+    * nvidia_gpu_sm20 - NVIDIA Fermi architecture
```

Comment:
Up above on line 35 it says:

> Special target values specific to Intel Processor Graphics support are accepted

That should be updated since the new targets you are adding are not specific to Intel processors.